defaults/main.yml

```diff
@@ -11,6 +11,10 @@ grafana_data:
   # Docker labels
   docker_labels: []
 
+grafana_volumes:
+  - "{{ grafana.grafana_storage }}:/var/lib/grafana"
+  - "{{ grafana.provisioning_dir }}:/etc/grafana/provisioning"
```

Comment:
Let's do it another way. Move vars from **grafana_data** to **grafana** and make a vars like:

```
grafana_storage: /var/grafana/storage
grafana_provisioning_dir: /var/grafana/provisioning
grafana_ldap_host_config_path: /var/grafana/config
```

etc.